src/pages/components/multiselect.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Multiselect
+description: >
+  Multiselect is a form component that allows multiple selections to be made from a list of
+  options. The options can be dynamically filtered based on what is typed.
+package: '@zendeskgarden/react-dropdowns'
+propSheets:
+  - dropdowns/src/elements/Multiselect/Multiselect.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To make multiple selections from a list of options</li>
+      <li>To filter a data set by keyword(s)</li>
```

Comment:
I'd suggest avoiding the parenthetical plural. See https://www.grammarphobia.com/blog/2015/04/parenthetical-plurals.html for context.